test/integration/test-released.js

```diff
@@ -49,8 +43,7 @@ function runTest(shouldKillPolyfillableApis) {
       });
     });
 
-    // There is really weird behavior when running this test in FF in
-    // saucelabs.
+    // There is really weird behavior when running this test in FF.
```

Comment:
Is this whole comment block (up to the TODO) still needed? In other files (i.e. `test/integration/test-amp-recaptcha-input.js`, comments regarding browser specific failures on Saucelabs have been removed entirely -- I assumed this is because these same issues are not guaranteed to persist/could be tooling-specific.

Reply:
Good catch. Removed the entire comment since it had to do with Sauce Labs.